test/obb/server/controllers/mock_request.clj

```diff
@@ -0,0 +1,12 @@
+(ns obb.server.controllers.mock-request
+  (:use clojure.test)
+  (:require [obb.server.controllers.index :as index]
+            [obb.server.core :as core]
+            [clojure.edn :as edn]
+            [ring.mock.request :as mock]))
+
+(defn http-get
+  "Mocks a HTTP GET request. Returns the body as edn"
+  [path]
+  (let [response (core/app (mock/request :get "/"))]
+    (assoc response :body (edn/read-string (:body response)))))
```

Comment:
Sounds good. There is something else needed, in the future we will have _authed_ requests. That can be an overload on the fn. And yes, this is not a mock, this is basically a networking bypass.

So, something like:

```clj
(defn parsed-response [http-method path & args])
```

And later we can add another for the auth, like `auth-parsed-response`.